servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java

```diff
@@ -115,6 +127,32 @@ public CertFailedRefreshNotificationTask(InstanceCertManager instanceCertManager
                 .collect(Collectors.toList());
     }
 
+    private List<X509CertRecord> getRecordsNotSnoozed(List<X509CertRecord> unrefreshedCerts) {
+        List<X509CertRecord> unsnoozedDomains = new ArrayList<>();
+        for (X509CertRecord x509CertRecord : unrefreshedCerts) {
+            String domainName = AthenzUtils.extractPrincipalDomainName(x509CertRecord.getService());
+            DomainData domainData = dataStore.getDomainData(domainName);
+            if (!isDomainSnoozed(domainData)) {
+                unsnoozedDomains.add(x509CertRecord);
+            }
+        }
+
+        return unsnoozedDomains;
+    }
+
+    private boolean isDomainSnoozed(DomainData domainData) {
+        Map<String, StringList> domainTags = domainData.getTags();
+        if (domainTags == null || domainTags.get(SNOOZED_DOMAIN_TAG_KEY) == null || domainTags.get(SNOOZED_DOMAIN_TAG_KEY).getList() == null) {
+            return false;
+        }
+
+        List<String> snoozeTagValues = domainTags.get(SNOOZED_DOMAIN_TAG_KEY).getList();
+        return snoozeTagValues.stream()
+                .filter(value -> value.toLowerCase().equals(SNOOZED_DOMAIN_TAG_VALUE))
+                .findAny()
+                .isPresent();
```

Comment:
can be replaced by `anyMatch(value -> value.toLowerCase().equals(SNOOZED_DOMAIN_TAG_VALUE))`

Reply:
Nice wasn't familiar with it thanks!